src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -55,9 +55,9 @@ class Crawler implements \Countable, \IteratorAggregate
     private $isHtml = true;
 
     /**
-     * @param mixed  $node     A Node to use as the base for the crawling
-     * @param string $uri      The current URI
-     * @param string $baseHref The base href value
+     * @param mixed|null  $node     A Node to use as the base for the crawling
+     * @param string|null $uri      The current URI
+     * @param string|null $baseHref The base href value
```

Comment:
I'm unsure what our CS rules are about that, but these changes feel a bit redundant… `mixed` can be literally anything, including `null`. And the other two parameters already default to `null`. 😕

Reply:
It can be null in the sense that it's optional, so I don't see why we should change the phpdocs.  Let's try to limit the changes to phpdocs to the bare minimum. We are spending too much time on that.

Reply:
it also allows passing null explicitly (being optional does not mean you always accept null)